### ISSUE TITLE
How to prevent truncation of floats?

### ISSUE THREAD

**Issue by Andrew-Max** [balderdashy/sails#3088]
I want to save a float to my database: 

```
39.267475553675148
```

Right now when I save this attribute it gets truncated to four places which is not precise enough for my needs.

Is there a way to prevent this truncation from happening?

**Comment by tjwebb** [comment]
What database are you using? This is likely specific to some adapter

**Comment by justsml** [comment]
You are hitting JS's native limits regarding decimal precision:
Simply run `console.log(39.267475553675148)`
Will lose you a digit: 39.26747555367515

**Comment by sailsbot** [comment]
Thanks for posting, @Andrew-Max.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
